src/components/MenuItems/MenuItems.tsx

```diff
@@ -12,7 +12,7 @@ export const MenuItems = ({ products }: Props): ReactElement => {
   }
   return (
     <div>
-      <div className="mb-12 grid grid-cols-3 px-16">
+      <div className="mb-12 mr-10 grid max-h-[60vh] grid-cols-3 overflow-auto px-16">
```

Comment:
Try to use precise measurements e.g mb-[20px], don't have to always if its like mb-1 or mt-2, but it is easier to read for larger measurements.